index.ts

```diff
@@ -164,34 +161,18 @@ async function main() {
   }
 }
 
-main();
+void (async () => {
+  try {
+    await main();
+  } catch (error) {
+    console.error(error);
+  }
+})();
```

Comment:
Wasn't 100% sure if this is the way to fix this lint error. ChatGPT says that the runtime will wait for this promise to complete before exiting the process.

Reply:
All right, let's use this approach.